### PR TITLE
Feature/7620 fix ssis publish

### PR DESCRIPTION
**Pull request by bjss-chris**
Activity: [AB#7620](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7620)
Task: [AB#7948](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7948)

### CODE REVIEW

File: NHSD.BuyingCatalogue.sln [hunk 1]
```diff
@@ -105,154 +105,234 @@ Project("{00D1A9C2-B5F0-4AF3-8072-F6C62B433612}") = "NHSD.BuyingCatalogue.Databa
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "NHSD.BuyingCatalogue.Database.Deployment", "src\NHSD.BuyingCatalogue.Database.Deployment\NHSD.BuyingCatalogue.Database.Deployment.csproj", "{20C95824-E0AD-488F-A903-BD0A9CABE847}"
 EndProject
+Project("{159641D6-6404-4A2A-AE62-294DE0FE8301}") = "NHSD.BuyingCatalogue.Database.PublishAll", "src\NHSD.BuyingCatalogue.Database.PublishAll\NHSD.BuyingCatalogue.Database.PublishAll.dtproj", "{7142A686-E318-4692-9C5E-DE74EC7873B9}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Development|Any CPU = Development|Any CPU
```
Comment (sam-roberts-bjss) [inline NHSD.BuyingCatalogue.sln:113]: Grateful if you could provide a brief comment just to outline the reason for the additional build configuration?

File: src/NHSD.BuyingCatalogue.Database.PublishAll/PublishAll.dtsx [hunk 2]
```diff
@@ -0,0 +1,4523 @@
+﻿<?xml version="1.0"?>
+<DTS:Executable xmlns:DTS="www.microsoft.com/SqlServer/Dts"
+  DTS:refId="Package"
+  DTS:CreationDate="12/12/2019 10:17:38 AM"
+  DTS:CreationName="Microsoft.Package"
+  DTS:CreatorComputerName="LPT3353"
+  DTS:CreatorName="BJSS\andy.berry"
```
Comment (sam-roberts-bjss) [inline src/NHSD.BuyingCatalogue.Database.PublishAll/PublishAll.dtsx:7]: Very minor & not really sure what to do but wondering whether or not we should keep this information in i.e. BJSS\andy.berry & LPT3353?

File: src/NHSD.BuyingCatalogue.Database.PublishAll/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+* AdditionalService
+* FrameworkSolutions
+* MarketingContact
+* SolutionCapability
+* SolutionEpic
+* SupplierServiceAssociation
+
+## Pre-requisites
+
+* Visual Studio 2019
```
Comment (sam-roberts-bjss) [inline src/NHSD.BuyingCatalogue.Database.PublishAll/README.md:17]: I assume we don't need a specific version of the .Net Framework?

File: src/NHSD.BuyingCatalogue.Database.PublishAll/README.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+3. Verify that data transfer has taken place and the destination database has been updated.
+4. Use a git client to revert your local changes.
+
+**DO NOT COMMIT PRODUCTION CREDENTIALS TO GIT.**
+
+## Deploying the package to Azure
+
+### Prerequisites
+
+The Azure-SSIS Integration Runtime has been provisioned  in Azure Data Factory as described [here](https://docs.microsoft.com/en-us/sql/integration-services/lift-shift/ssis-azure-lift-shift-ssis-packages-overview?view=sql-server-ver15#provision-ssis-on-azure).
```
Comment (sam-roberts-bjss) [inline src/NHSD.BuyingCatalogue.Database.PublishAll/README.md:63]: Spacing?

File: NHSD.BuyingCatalogue.sln (hunk 1 above)
Comment (bjss-chris) [reply NHSD.BuyingCatalogue.sln:113]: The default configuration for SSIS is `Development`. Therefore, the development configuration was created automatically when I added the existing SSIS project. Given that it doesn't have any impact on the existing build setup I decided to leave as is.

In most cases there isn't any reason to have separate debug and release configurations for SSIS so (other than reducing the number of configurations) there would be no benefit in consolidating them.

File: src/NHSD.BuyingCatalogue.Database.PublishAll/PublishAll.dtsx (hunk 2 above)
Comment (bjss-chris) [reply src/NHSD.BuyingCatalogue.Database.PublishAll/PublishAll.dtsx:7]: Removed.

File: src/NHSD.BuyingCatalogue.Database.PublishAll/README.md (hunk 3 above)
Comment (bjss-chris) [reply src/NHSD.BuyingCatalogue.Database.PublishAll/README.md:19]: I've done a bit of hunting around and I'm struggling to find anywhere that Microsoft lists a specific .NET Framework requirement.

.NET is required for SSIS but I can only assume that the installation steps outlined in the readme must make the sure the necessary framework is present and/or install if not.

Comment (sam-roberts-bjss) [reply src/NHSD.BuyingCatalogue.Database.PublishAll/README.md:19]: I think the .Net Framework is installed as part of Visual Studio anyway. So what you've outlined is fine. See how the team get on once this is merged in.

**Pull request merged**